gocd/bci_repo_publish.py

```diff
@@ -153,7 +173,7 @@ def run(self, version, token=None):
             return
 
         # Trigger publishing
-        if token is None:
+        if repo_token is None or rebuild_token is None:
```

Comment:
why this change? I think rebuild_token can still be unset to avoid the rebuild?

Reply:
I have modified it so that either of the token triggered steps can be skipped